client/client.go

```diff
@@ -169,7 +175,7 @@ func (c *Client) MakeRestRequest(method string, path string, body *container.Con
 // Authenticate is used to
 func (c *Client) Authenticate() error {
 	method := "POST"
-	path := "/api/v1/auth/login"
+	path := "/login"
 	if c.platform == "nd" {
 		c.domain = "local"
```

Comment:
We cannot change this one as this would break on non ND mso we need to do it this way:
```
method := "POST"
	path := "/api/v1/auth/login"
	if c.platform == "nd" {
		c.domain = "local"
                 path = "/login"
```